test/trinity/BorrowerOperations_FeesTest.js

```diff
@@ -196,6 +207,43 @@ contract("BorrowerOperations_Fees", async accounts => {
         })
     })
 
+    describe('closeVessel', () => {
+        it('does not add fees mid epoch', async () => {
+            const {erc20, debtToken, borrowerOperations} = contracts.core
+            await openVessel(alice)
+            await openVessel(bob)
+            await skipToNextEpoch()
+
+            await borrowerOperations.collectVesselFee(erc20.address, alice)
+            
+            const treasuryBalanceBefore = await debtToken.balanceOf(treasury)
+
+            await mintDebtTokens(alice, vesselTotalDebt)
+            await closeVessel(alice)
+
+            const treasuryBalanceAfter = await debtToken.balanceOf(treasury)
+
+            assert.isTrue(treasuryBalanceBefore.eq(treasuryBalanceAfter))
```

Comment:
I would prefer us to use assert.equal(A, B), so the errors will be more descriptive and better than: True is not False

Reply:
assert.equal matcher does not work on the BN type that they use

Reply:
assert.equal(before.toString(), after.toString())